src/specular/stages/batches.rs

```diff
@@ -183,21 +182,44 @@ fn decode_batches(
     batcher_ransaction: &SpecularBatcherTransaction,
     state: &RwLock<State>,
 ) -> Result<Vec<SpecularBatchV0>> {
```

Comment:
can you document the semantics; also nit: `batcher_transaction` typo

Reply:
done